src/test/java/fiori/screens/ServiceOrderQuotationScreen.java

```diff
@@ -0,0 +1,37 @@
+package fiori.screens;
+
+import eu.ibagroup.junase.web.test.WebDriverManager;
+import eu.ibagroup.junase.web.util.Wait;
+import fiori.component.Input;
+import fiori.component.TabContainer;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+public class ServiceOrderQuotationScreen {
+
+    private static final String INPUT_BY_LABEL_XPATH = "//label[contains(text(),'%s')]//following::input[1]";
+
+    private static final String APPLICATION_IFRAME_ID = "application-ServiceQuotation-create";
+
+    private final TabContainer tabContainer = new TabContainer(By.id("anchorBar"));
+
+    public Input setInputByLabel(String labelName, String value) {
```

Comment:
the method should be void

Reply:
public void setInputByLabel(String labelName, String value) {
        new Input(By.xpath(String.format(INPUT_BY_LABEL_XPATH, labelName))).setText(value);
    }
    
     public void setInputByLabelAndClickEnter(String labelName, String value) {
        Input input = new Input(By.xpath(String.format(INPUT_BY_LABEL_XPATH, labelName)))
        input.setText(value)
        input.sendKeys(Keys.ENTER);
    }